modules/administration-guide/pages/configuring-workspace-target-namespace.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: CONCEPT
+:navtitle: Configuring workspace target {orch-namespace}
+:keywords: administration guide, configuring, namespace
+:page-aliases: installation-guide:configuring-namespace-strategies, configuring-namespace-strategies
+
+[id="configuring-workspace-target-namespace_{context}"]
+= Configuring workspace target {orch-namespace}
+
+For each user, {prod-short} isolates the workspaces in a {orch-namespace}.
+You can configure the {orch-namespace} name template.
+
+{prod-short} server checks usernames and IDs for compatibility with {orch-name} objects naming convention before creating a {orch-namespace} from a template:
+
+*  Incompatible usernames or IDs are reduced to the nearest valid name by replacing groups of unsuitable symbols with the `-` symbol. 
+* The addition of a random 6-symbol suffix prevents IDs from collisions. The result is stored in preferences for reuse.
+* {kubernetes} limits the length of a {orch-namespace} name to 63 characters (this includes the evaluated placeholders). Additionally, the names (after placeholder evaluation) must be valid DNS names.
+* OpenShift limits the length further to 49 characters.
+* The `_<userid>_` placeholder is evaluated into a 36 character long UUID string.
```

Comment:
Interestingly so we do really support the 36 chars postfix in the namespace names? was it verified?

Reply:
in any case, as a user / admin, I could not imagine having fun with namespace setup like `dev-26c52e9d-780d-42d2-9481-869357844e53`

Reply:
Che on Minikube: `<userid>-aka-<username>-namespace-for-che` translating to

`cgezegvsb2nhba-aka-user3-namespace-for-che-6m2w2b`